src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,127 +1,96 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
-    /**
-     * Верните текущую дату в виде строки в зависимости от запроса.
-     *
-     * @param datePart Запрос на часть даты или всю дата целиком:
-     *                 - FULL - текущая дата целиком год, месяц, день (число месяца)
-     *                 в формате YYYY-MM-DD, возвращаемое значение по умолчанию;
-     *                 - YEAR - текущий год;
-     *                 - MONTH - название текущего месяца;
-     *                 - DAY - текущий день (число месяца);
-     *                 В любом другом случае бросить DateTimeException
-     **/
+
+    private static final ZoneOffset UKRAINE_TIMEZONE_OFFSET = ZoneOffset.of("+02:00");
+    private static final LocalDate today = LocalDate.now();
```

Comment:
let's use local variables in each method instead
imagine that your application is constantly running
this variable would be created only once

Reply:
done